src/analyzer/engines/gensim.py

```diff
@@ -0,0 +1,71 @@
+from src.analyzer.engines.engine import ModelingEngine
+
+from gensim.models.coherencemodel import CoherenceModel
+from gensim.models.ldamodel import LdaModel
+from gensim.models.tfidfmodel import TfidfModel
+from gensim.models import Nmf
+
+class GensimEngine(ModelingEngine):
+    """
+    GensimEngine implements the methods for fitting and evaluating gensim models on a
+    corpus.
+    """
+    def __init__(self, min_topics, max_topics):
```

Comment:
Could we put in some reasonable default values for `min_topics` and `max_topics` and turn these into kwargs rather than positionals?  Maybe `def __init__(self, min_topics=10, max_topics=20)`?